docs/documentation/features/scenes.md

```diff
@@ -0,0 +1,113 @@
+---
+layout: documentation
+---
+
+{% include base.html %}
+
+# Scenes
+
+In the general context of home automation, a scene is a defined set of states of one or more home devices. For example, a **night scene**, which turns on the indoor lights after sunset. The notion of scenes is not directly available in Eclipse Smarthome but scenes can be created by the means of using rules.
```

Comment:
I like the example. Minor comment: the text "which turns on the indoor lights after sunset" describes a rule with a trigger ("sunset"). The scene itself is simply "indoor lights are turned on".

Suggestion for improvement: For example, a night scene defined by 'all indoor lights are turned on' could be used to turn on indoor lights after sunset.